### PR TITLE
Rename accordion module

### PR DESCRIPTION
**Pull request by OsamaAtReloadly**

### CODE REVIEW

**Comment by iSeremet-Reloadly** [comment]
Hey @OsamaAtReloadly do you see anything wrong with this PR?

File: src/lib/components/modal/reloadly-modal-container/reloadly-modal-container.component.html
Comment (SabreFoxx-Reloadly) [inline src/lib/components/modal/reloadly-modal-container/reloadly-modal-container.component.html:1]: I already made these changes. I think there's something wrong with the PR

**Comment by salimzakari-reloadly** [comment]
You need to sync your branch with main @OsamaAtReloadly

**Comment by farouk-reloadly** [comment]
The PR is supposed to be into `develop` not `main` 👀

**Review by iSeremet-Reloadly — CHANGES_REQUESTED** [review]
!

**Review by OsamaAtReloadly — COMMENTED** [review]
@iSeremet-Reloadly Sorry I resynced with develop and change the destination branch.

**Review by iSeremet-Reloadly — APPROVED** [review]

**Comment by farouk-reloadly** [comment]
@OsamaAtReloadly  you have to also increase the package version otherwise the build won't be successful

**Review by salimzakari-reloadly — APPROVED** [review]

**Review by iSeremet-Reloadly — APPROVED** [review]

**Comment by OsamaAtReloadly** [comment]
> @OsamaAtReloadly you have to also increase the package version otherwise the build won't be successful

Updated now.

**Review by SabreFoxx-Reloadly — APPROVED** [review]

**Pull request merged**